### PR TITLE
refactor: stop releasing edge edition

### PR DESCRIPTION
**Pull request by zmstone**
Prior to EMQX 5.0, the edge edition's main difference comaring
to standard edition are:
* Less number of plugins in the release (smaller package size)
* Smaller number is vm.args (for lower memory usage)

Starting from 5.0 most of the plugins are included as features,
the tuned vm.args arguments does not justify a special release edition.

Also as nanomq is getting mature,
EMQ's recommendtation for MQTT broker in edge is https://nanomq.io/

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -168,6 +166,7 @@ defmodule EMQXUmbrella.MixProject do
       compiler: :permanent,
       runtime_tools: :permanent,
       redbug: :permanent,
+      xmerl: :permanent,
```
Comment (thalesmg) [inline mix.exs:169]: This changes the order of application loading/starting.

Perhaps keep it at the end like the rebar version?

I think the CI check missed this and the license/enterprise_conf problem.  Trying to fix it here: https://github.com/emqx/emqx/pull/8058

File: rebar.config.erl [hunk 2]
```diff
@@ -330,6 +308,7 @@ relx_apps(ReleaseType, Edition) ->
         compiler,
         runtime_tools,
         redbug,
+        xmerl,
```
Comment (thalesmg) [inline rebar.config.erl:311]: Q: is the intention to change the order of starting/loading this indeed?  It was one of the last ones before.

Comment (zmstone) [reply rebar.config.erl:311]: this application can be started in any order.

**Review by thalesmg — APPROVED** [review]

File: mix.exs (hunk 1 above)
Comment (zmstone) [reply mix.exs:169]: I changed both sides, so the CI won't complain (not for this app).

**Pull request merged**